planet.js

```diff
@@ -606,7 +606,7 @@ const _connectRoom = async roomName => {
       });
       await channelConnection.setMicrophoneMediaStream(mediaStream);
     };
-    _latchMediaStream(); */
+    _latchMediaStream(); 
```

Comment:
This should probably reflect clicking/unclicking the microphone button rather than the multiplayer connect button.